src/components/HomePage/Movie/Movie.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from "prop-types";
+import styles from './Movie.css'
+
+const Movie = ({ film }) => {
+    return (
+      <div>
+        <img src={film.picture} className={styles.movie} />
+        <div className={styles.movies__name}>
+          <span>{film.name}</span>
+          <div className={styles.movies__year}>{film.year}</div>
+        </div>
+        <div className={styles.movies__description}>{film.description}</div>
+      </div>
+    );
+}
+
+Movie.propTypes = {
+  film: PropTypes.object
```

Comment:
it will be better to be more specific, so you need to define `film` properties

Reply:
https://github.com/Azaliyaa/netflix-roulette/pull/1/commits/c9362e737f87aed20a979cc23bf00d54e63dd86d